README.md

```diff
@@ -228,3 +228,36 @@ Including the line
 ```
 in the relevant package `BuildFile.xml` allows for
 including our headers in the code of that package.
+
+## Running LST in a CVMFS-less setup
+
+The setup scripts included in this repository assume that the [CernVM File System (CVMFS)](https://cernvm.cern.ch/fs/) is installed. This provides a convenient way to fetch the required dependencies, but it is not necessary to run LST in standalone mode. Here we briefly describe how to build and run it when CVMFS is not available.
+
+The necessary dependencies are CUDA, ROOT, the Boost libraries, Alpaka, and some CMSSW headers. CUDA, ROOT, and Boost, are fairly standard libraries and are available from multiple package managers. You will need to clone the [Alpaka](https://github.com/alpaka-group/alpaka) and [CMSSW](https://github.com/cms-sw/cmssw) repositories. Then all that is left to do is set some environment variables. We give an example of how to do this in lnx7188/cgpu-1.
```

Comment:
I was hoping for something more specific than "clone CMSSW", given its size:
![image](https://github.com/SegmentLinking/TrackLooper/assets/15358704/c28ce4f8-03ac-49ea-add6-9341d88011d3)

Reply:
How about doing this? It only ends up being 10 MB.
```
git clone --filter=blob:none --no-checkout --depth 1 --sparse https://github.com/cms-sw/cmssw
cd cmssw
git sparse-checkout add HeterogeneousCore/AlpakaInterface
git checkout
cd ..
du -sh cmssw
# 10M	cmssw
```

Reply:
Sounds good, thanks! Please push it and I merge.